ccd-definition/ComplexTypes/CareSupervision/StandardDirectionOrder.json

```diff
@@ -66,5 +66,13 @@
     "FieldType": "JudgeAndLegalAdvisor",
     "ElementLabel": "Judge and Justices' Legal Adviser",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirectionOrder",
+    "ListElementCode": "removalReason",
+    "FieldType": "TextArea",
```

Comment:
are you adding this here for nikhils story or what's it for?

Reply:
No, should not be here. Will remove.